source/agora/consensus/Fee.d

```diff
@@ -435,29 +435,31 @@ public class FeeManager
 
     /***************************************************************************
 
-        Calculate the transaction fee and adjust the fee based on the
-        transaction's size measured in bytes.
+        Calculate the transaction fee rate.
 
-        The bigger the transaction size is, the smaller the the fee becomes.
+        This calculates the fee rate of the transaction, and can be used to
+        compare two transactions for inclusion in a block / pool eviction.
+        If the fee is not an exact multiplicator of the tx size,
+        the remainder will be ignored.
 
         Params:
-            tx = transaction for which we want to calculate the adjusted fee
-            peekUTXO = UTXO finder
-            tot_fee = adjusted fee
+            tx = transaction for which we want to calculate the fee rate
+            peekUTXO = UTXO finder (with or without replay protection)
+            rate = This transaction minimum fee rate
```

Comment:
```suggestion
            rate = This transaction's fee rate
```

Reply:
I know we are ignoring the remainder but it is misleading when we have `minimum fee rate` as `Consensus Param`

Reply:
Right, should be better worded.